src/Lib/Elastic/Filter/Params/FunctionalNamesParams.php

```diff
@@ -19,18 +19,22 @@
  *
  * @package SphereMall\MS\Lib\Elastic\Filter\Params
  */
-class FunctionalNamesParams implements ElasticParamElementInterface, ElasticParamBuilderInterface
+class FunctionalNamesParams extends BasicParams implements ElasticParamElementInterface, ElasticParamBuilderInterface
```

Comment:
BrandsParams, EntityGroupsParams and FunctionalNamesParams classes looks almost the same except type and field